### PR TITLE
Regenerated api from updated swagger spec

### PR DESCRIPTION
**Pull request by skiselev1**
# Description

This PR includes the following proposed change(s):

- Regenerated api from updated swagger spec, updated Application form to call a new api methods to retrieve lookup information, such as: provinces and so on

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [x] No

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/csrs-portal/nginx.conf [hunk 1]
```diff
@@ -35,6 +35,11 @@ location /nginx_status {
     access_log off;
 }
 
+# not sure, what url should be added in proxy_pass
```
Comment (pbolduc) [inline src/frontend/csrs-portal/nginx.conf:38]: Why are we adding commented out lines?

File: src/frontend/csrs-portal/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "ng": "ng",
     "configure": "ts-node setenv.ts",
-    "start": "yarn run configure -- --environment=dev && ng serve",
+    "start": "yarn run configure -- --environment=dev && ng serve --proxy-config proxy.conf.json",
```
Comment (pbolduc) [inline src/frontend/csrs-portal/package.json:7]: this works, but you could also add another script, does everyone have to run with proxy?

File: src/frontend/csrs-portal/proxy.conf.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "/api/*": {
+      "target": "http://jag-csrs-portal-public-f0b5b6-dev.apps.silver.devops.gov.bc.ca/",//"http://localhost:8081/",
```
Comment (pbolduc) [inline src/frontend/csrs-portal/proxy.conf.json:3]: confused, why are you proxying to the remote dev portal?

File: src/frontend/csrs-portal/nginx.conf (hunk 1 above)
Comment (skiselev1) [reply src/frontend/csrs-portal/nginx.conf]: Commented lines were deleted.

File: src/frontend/csrs-portal/package.json (hunk 2 above)
Comment (skiselev1) [reply src/frontend/csrs-portal/package.json]: It was added a new script start-proxy.

File: src/frontend/csrs-portal/proxy.conf.json (hunk 3 above)
Comment (skiselev1) [reply src/frontend/csrs-portal/proxy.conf.json]: Remote dev portal was replaced by local.

**Comment by pbolduc** [comment]
Please delete all the files in the api and model folders that contain generated files. you will noticed when you regenerate, there are a number of deleted files cause the file names or models have changed names.

```
        deleted:    api/portalAccount.service.ts
        deleted:    api/portalDocument.service.ts
        deleted:    api/portalFeedback.service.ts
        deleted:    api/portalFile.service.ts
        deleted:    api/portalMessage.service.ts
        deleted:    api/portalUserRequest.service.ts
        deleted:    model/portalAccount.model.ts
        deleted:    model/portalDocument.model.ts
        deleted:    model/portalFeedback.model.ts
        deleted:    model/portalFileToList.model.ts
        deleted:    model/portalMessage.model.ts
        deleted:    model/portalUserRequest.model.ts
```

**Review by pbolduc — APPROVED** [review]

**Pull request merged**